server/docs/api/swagger.yml

```diff
@@ -172,6 +172,28 @@ paths:
           description: "Invalid input or other error"
         404:
           description: "Application not found"
+  /forests/{id}/regulations:
```

Comment:
isn't there a `forests` endpoint that was also created?

Reply:
@lpendem which story?

Reply:
it was done with #5